setup.md

```diff
@@ -10,9 +10,10 @@ You need to download and install OpenRefine and download a data file to follow t
 
 ### Downloading OpenRefine
 
-You can download OpenRefine from [http://openrefine.org/download.html](http://openrefine.org/download.html). This lesson has been tested with all versions of OpenRefine, from 2.8 to 3.1.  
+You can download OpenRefine from [http://openrefine.org/download.html](http://openrefine.org/download.html). This lesson has been tested with all versions of OpenRefine up to 3.1.  
 
-OpenRefine 3.1 is recommended.  
+OpenRefine 3.1 is recommended. 
+If using an older version, it is recommended to upgrade to the tested version (currently 3.1). 
```

Comment:
suggest we use "it is recommended you upgrade to the latest tested version."

Reasons:

* We use "you" throughout the lesson to address actions etc.
* We've already said the version in the previous sentence - omitting it here means we only have one place to make a change as we update the latest tested version
* Need to be clear we mean the latest tested version not just any tested version

Reply:
Accepted. I saw that too and should have applied it to my changes,